src/kvazaarCMake.pc.in

```diff
@@ -1,6 +1,6 @@
 prefix=@CMAKE_INSTALL_PREFIX@
 exec_prefix=${prefix}
-libdir=${prefix}/@CMAKE_INSTALL_LIBDIR@
+libdir=@CMAKE_INSTALL_FULL_LIBDIR@
```

Comment:
Ah, here you actually _don't want_ an absolute path always in order to enable a relocatable package...

Instead, one has to check if `CMAKE_INSTALL_LIBDIR` was absolute and act accordingly. See for example https://github.com/lensfun/lensfun/commit/542b9894b5318878444b147d2927f98d3587e8e6

Reply:
And while you're doing that, you'll realize [there's no need for two separate .pc.in templates, you're setting a couple of placeholder variables anyway](https://github.com/ultravideo/kvazaar/issues/381#issuecomment-1898211561).

Reply:
Considering that the kvazaarCMake.pc.in is based on the one that is used by uvg266, we are going to be maintaining two templates regardless, unless automake can also use the CMake version. Additionally, the template has been updated literally once, nine years ago, it is not really an issue to have two templates, even if it is not an optimal solution. 

If you are adamant on us not having two templates on this repository I will accept a PR, but please submit a similar PR for uvg266 also.

Reply:
> If you are adamant on us not having two templates on this repository

I'm not. Just sharing what has become sort of "best practice" I've experienced across several imaging related libraries, especially ones that are (or were) supporting both autotools and cmake, like libtiff and libheif for example. I see no reason not to reuse it...

It's of course your prerogative how you want to maintain your repositories, and apologies to have annoyed you with my comments.

Reply:
Yes, I understand why having two templates cannot be considered a best practice. However, considering the low maintenance needed for these files, and the fact that we aim not to support both autotools and cmake, but replace autotools with cmake, I'm fine with not following best practices here.

For your other suggestions I'm grateful, as I have very little experience with CMake. 

> apologies to have annoyed you with my comments.

I interpreted your usage of ellipsis as a criticism towards the fact that I was not aware of the best practices, which I assume was not the intention? I would like to apologize for my maybe overly harsh comments towards you. 

Otherwise, in your opinion, does the current state of the PR adhere to the best practices?

Reply:
No problem. There's just two more minor details, I'll leave comments on the latest commits.